tools/clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -2030,6 +2030,7 @@ std::string CompilerInvocation::getModuleHash() const {
 #define ENUM_LANGOPT(Name, Type, Bits, Default, Description) \
   code = hash_combine(code, static_cast<unsigned>(LangOpts->get##Name()));
 #define BENIGN_LANGOPT(Name, Bits, Default, Description)
+#define BENIGN_LANGOPT_BOOL(Name, Default, Description)
```

Comment:
LANGOPT_BOOL is not given a default define in LnagOptions.fixed.def. Shouldn't it have to be defined here? Perhaps this is dead code for our builds?

Reply:
Added default define for LANGOPT_BOOL.